_docs_integrate/26-requests-over-messages.md

```diff
@@ -0,0 +1,143 @@
+
+```jsonc
+{
+    "recipients": ["<the address of the Peer Connector>"],
+    "content": {
+        // the content you copied in the step before
+    }
+}
+```
+
+This is where the automation of the Enmeshed Runtime steps in and moves the Request from status `Draft` to status `Open`. You can observe this behaviour by querying the Request via `GET /api/v2/Requests/Outgoing/{id}` on the Sender Connector.
+
+## Fetch the Request
+
+The Peer will receive the Request as a Message. Therefore you have to synchronize the Peer Connector (`GET /api/v2/Account/Sync`).
+
+The Enmeshed Runtime will read the Message and create a new incoming Request. You can observe this by long polling the incoming Requests or by waiting for the `consumption.incomingRequestReceived` event.
+
+The long polling is done by calling the `GET /api/v2/Requests/Incoming` route. You can use the query params `source.reference=<id-of-the-message>` and `status=ManualDecisionRequired` to filter only for Requests that belong to the Message that contained the Request.
+
+For more information about the events you can head over to the [Connector Modules site]({% link _docs_integrate/03-connector-modules.md %}) and read about the [AMQP Publisher module]({% link _docs_integrate/03-connector-modules.md %}#amqppublisher) and the [WebhooksV2 module]({% link _docs_integrate/03-connector-modules.md %}#webhooksv2) that are propagating events.
+
+{% include copy-notice description="After you received the Request, save its `id` for the next step." %}
+
+## Answer the Request
+
+### Accept
+
+If you want to accept the Request you can do so by calling the `POST /api/v2/Requests/Incoming/{id}/Accept` route. You can use the `id` you saved in the previous step. In the payload you have to accept at least all RequestItems where the `mustBeAccepted` property is set to `true`. In case of the example Request the payload is the following:
+
+```jsonc
+{
+    "items": [
+        {
+            "accept": true
+        }
+    ]
+}
+```
+
+### Reject
+
+If you want to reject the Request you can do so by calling the `POST /api/v2/Requests/Incoming/{id}/Reject` route. You can use the `id` you saved in the previous step. In the payload you have to reject all RequestItems. In case of the example Request the payload is the following:
+
+```jsonc
+{
+    "items": [
+        {
+            "accept": false
+        }
+    ]
+}
+```
+
+### Runtime automation
+
+In the response you can see the Request has moved to status `Decided`. This is where the Enmeshed Runtime steps in and handles the Request based on you decision. The Enmeshed Runtime will move the Request to status `Completed` and send the Response to the Sender via a Message. This behavior can be observed by querying the Request again after a short waiting time (`GET /api/v2/Requests/Incoming/{id}`).
```

Comment:
> In the response you can see the Request has moved to status `Decided`

No matter if you accepted or rejected the Request: the response is similar. You can see that the Request moved to status `Decided`.